Software/Frontend/app/src/main/java/hr/foi/air/baufind/service/JobService/JobService.kt

```diff
@@ -0,0 +1,48 @@
+package hr.foi.air.baufind.service.JobService
+
+import android.util.Log
+import hr.foi.air.baufind.ws.network.NetworkService
+import hr.foi.air.baufind.ws.network.TokenProvider
+import hr.foi.air.baufind.ws.request.JobCreateBody
+import java.util.Base64
+
+class JobService(){
+    suspend fun addJobAsync(jobDao: JobDao, tokenProvider: TokenProvider): JobResponse {
+        val service = NetworkService.createJobService(tokenProvider)
+
+        val jobBody = JobCreateBody(
+            title = jobDao.name,
+            description = jobDao.description,
+            allow_worker_invite = jobDao.allowInvitations,
+            skills = jobDao.positions,
+            pictures = jobDao.images.map{ Base64.getEncoder().encodeToString(it)},
+            latitude = jobDao.lat,
+            longitude = jobDao.long,
+            location = "Fiksni zasad"
```

Comment:
To što si spomenuo ostavi sada kako je, svakako ćemo se dogovoriti za idući sprint onda kako ćemo točno jer ćemo tad raditi modularnost. Možda da i unos teksta lokacije bude u modulu kao pa modul ne samo da služi za lat i lng, nego i za baš lokaciju? Onda bi se on mogao spajati na npr. neki API za dohvaćanje naziva lokacije iz koordinata i obrnuto, no ovo je samo ideja.